content/en/docs/concepts/scheduling-eviction/assign-pod-node.md

```diff
@@ -271,6 +271,17 @@ If omitted or empty, it defaults to the namespace of the pod where the affinity/
 All `matchExpressions` associated with `requiredDuringSchedulingIgnoredDuringExecution` affinity and anti-affinity
 must be satisfied for the pod to be scheduled onto a node.
 
+#### Namespace Selector
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+Users can also select namespaces using `namespaceSelector`, which is a label query over the set of namespaces.
+The affinity term is applied to the union of the namespaces selected by `namespaceSelector` and the ones listed in the `namespaces` field.
+Note that an empty `namespaceSelector` ({}) matches all namespaces.
+
+This feature is currently alpha and disabled by default, and can be enabled via 
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+`PodAffinityNamespaceSelector` in both kube-apiserver and kube-scheduler.
```

Comment:
Please add this feature gate into https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ and provide a brief description.

Reply:
updated.